eclair-node/src/main/scala/fr/acinq/eclair/Plugin.scala

```diff
@@ -29,17 +31,53 @@ trait Plugin {
 
 }
 
-object Plugin {
+object Plugin extends Logging {
+
+  val versionRegex = """^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)([-SNAPSHOT]*)""".r
```

Comment:
This is a bit fragile, and doesn't match the version 0.3 that we have released last year.